app/scripts/views/mixins/signin-mixin.js

```diff
@@ -138,6 +138,10 @@ define(function (require, exports, module) {
 
       return this.invokeBrokerMethod(brokerMethod, account)
         .then(this.navigate.bind(this, this.model.get('redirectTo') || 'settings', {}, navigateData));
+    },
+
+    getCurrentQueryString () {
```

Comment:
We call `Query` [Search](https://github.com/mozilla/fxa-content-server/blob/9cbf5d91dea1e871251ffccdbdda1db6d06cb2c8/app/scripts/views/base.js#L938) in other functions, can you use it here too? And remove the `Current`. You might as well just move this function to views/base.js with the others so it's available everywhere.

Pedantic as it is, this function also needs tests wherever it lands.